index.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-alert */
+/* eslint-disable max-classes-per-file */
+class Book {
```

Comment:
- [ ] Please remove disabled linters as the issue can be solved.

- [ ] `/* eslint-disable no-use-before-define */` because you are using **BookList** in line 13 before it's was defined, that means `class BookList {
  static getList() {
    let books;
    if (localStorage.getItem('books') === null) {
      books = [];
    } else {
      books = JSON.parse(localStorage.getItem('books'));
    }
    return books;
  }

  static addItem(book) {
    const books = BookList.getList();
    books.push(book);
    localStorage.setItem('books', JSON.stringify(books));
  }

  static deleteItem(title) {
    const books = BookList.getList();
    books.forEach((book, index) => {
      if (book.title === title) {
        books.splice(index, 1);
      }
    });
    localStorage.setItem('books', JSON.stringify(books));
  }
}` should be before it.

- [ ] `/* eslint-disable no-alert */` because you are using alerts in add book and remove the book, and evenlisnter form.